lua/focus/modules/functions.lua

```diff
@@ -8,6 +8,7 @@ M.focus_enable = function()
     end
 
     vim.g.focus_disable = false
+    require('focus.modules.resizer').goal = 'autoresize'
```

Comment:
Can't we pass that down as an argument to the `resize()` function?

Reply:
Also rename `goal` to `action` and have the default action set to `autoresize`?

Reply:
good idea! I'll make that change now.